backend/entityservice/integrationtests/dbtests/test_insertions.py

```diff
@@ -88,12 +88,14 @@ def test_insert_many_clks(self):
         assert elapsed_time < 2
 
         stored_encoding_ids = list(get_encodingblock_ids(conn, dp_id, '1'))
-        fetch_time = time.perf_counter() - end_time
+        id_fetch_time = time.perf_counter() - end_time
         # retrieval of encoding ids should be much faster than insertion
-        assert fetch_time < elapsed_time
-
+        assert id_fetch_time < elapsed_time
```

Comment:
these timing assertions might not always hold. (e.g. if there is a load spike in the db) So should we really fail if it goes wrong once?